GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesListController.cs

```diff
@@ -71,13 +71,13 @@ public bool IsValidGitWorkingDir(string path)
             };
 
             var repositories = ThreadHelper.JoinableTaskFactory.Run(() => RepositoryHistoryManager.Locals.LoadRecentHistoryAsync());
-            splitter.SplitRecentRepos(repositories, mostRecentRepos, lessRecentRepos);
+            splitter.SplitRecentRepos(repositories, mostRecentRepos, lessRecentRepos, null, null, null);
```

Comment:
What are these `null`s? Add "argName: " if these cannot be turned into default arguments.